config.yaml

```diff
@@ -384,33 +384,33 @@ Facebook:
 #  notes: ''
 #  strike_out: No
 
-#Google:
-#  career_page:
-#  office_location:
-#  office_picture:
-#  glassdoor:
-#    link:
-#    # Overwrites
-#    rating:
-#    company_size:
-#    company_revenue:
-#  software_engineer:
-#    link:
-#    # Overwrites
-#    salary:
-#    bonus:
-#    stock_options:
-#  pantry:
-#  annual_leaves:
-#  flexible_hours:
-#  benefits:
-#    good_insurance:
-#    pregnancy:
-#    maternity_leaves:
-#    covers_dependents:
-#    extras:
-#  notes: ''
-#  strike_out: No
+Google:
+ career_page: https://careers.google.com/
+ office_location: Mapletree Business City
+ office_picture: pictures/google.png
+ glassdoor: 
+   link: https://www.glassdoor.com/Reviews/Google-Reviews-E9079.htm
+   # Overwrites
+   rating: 4.4
+   company_size: 10k+
+   company_revenue: $10B+
+ software_engineer: 
+   link: https://www.glassdoor.com/Salary/Google-Software-Engineer-Singapore-Salaries-EJI_IE9079.0,6_KO7,24_IL.25,34_IN217.htm
+   # Overwrites
+   salary: $8000-9000
```

Comment:
In the link, I think it's about $8625. Then there's additional $16k. So we can do $8625 - $10k

Reply:
$8625 - $10000